meta-ibm/recipes-phosphor/inventory/phosphor-inventory-manager/obmc-clear-all-faults@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Clears all fault LED to %i
+Wants=xyz.openbmc_project.Inventory.Manager
+After=xyz.openbmc_project.Inventory.Manager
+After=op-reset-chassis-running@0.service
+ConditionPathExists=!/run/openbmc/chassis@0-on
+
+[Service]
+Type=oneshot
+Restart=no
+EnvironmentFile={envfiledir}/obmc/led/clear-all/faults/config
+ExecStart=/bin/bash -c "clear-all-faults.sh %i ${{EXCLUDED_GROUPS}}"
```

Comment:
Is clear-all-faults.sh already a part of the flash image?

Reply:
yes , it is put into /usr/bin by automake change in https://github.com/ibm-openbmc/phosphor-inventory-manager/pull/1/files